lib/components/GlobalHeader/GlobalSearch.tsx

```diff
@@ -0,0 +1,64 @@
+import { MagnifyingGlassIcon } from '@navikt/aksel-icons';
+import { useState } from 'react';
+import { DsSearch, useRootContext } from '../';
+import { useIsDesktop } from '../../hooks/useIsDesktop';
+import styles from './globalSearch.module.css';
+
+export interface GlobalSearchProps {
+  onSearch: (query: string) => void;
+}
+
+export const GlobalSearch = ({ onSearch }: GlobalSearchProps) => {
+  const isDesktop = useIsDesktop();
+  const [query, setQuery] = useState('');
+  const { languageCode, closeAll } = useRootContext();
+  const { search, placeholder } = getTexts(languageCode);
+  const onSearchbarEnter = () => {
+    onSearch(query);
+    closeAll();
+  };
+  return (
+    <form
+      onSubmit={(e) => {
+        e.preventDefault();
+        onSearchbarEnter();
+      }}
+    >
+      <DsSearch className={styles.globalSearch} data-size={isDesktop ? 'lg' : 'md'}>
```

Comment:
Is the idea that this search field will be an unassisted input, and later shift on to autocomplete? 

Have you considered using the <Searchbar /> here?

Reply:
Yes, in a meeting between InfoPortal, the Innbox team, and Access Management, and then elaborated on in the subsequent Components sync meetings, it was decided that the global search will just reroute the user to the open pages search for now. InfoPortal is in charge of the Global Search functionality, but the functionality won't be ready for autocomplete in all of our solutions for December, so that will have to come later. For now, we will all just have to reroute to their page with the search query the user enters into this very simple search field.